tests/e2e/framework/kubernetes.go

```diff
@@ -793,15 +795,29 @@ EOF`, k.KubeConfig, dummyValidationRule)
 	return nil
 }
 
-func (k *KubeInfo) deployIstioWithHelm() error {
-	yamlFileName := filepath.Join(istioInstallDir, helmInstallerName, "istio", "templates", "crds.yaml")
+func (k *KubeInfo) deployCRDs(kubernetesCRD string) error {
+	yamlFileName := filepath.Join(istioInstallDir, helmInstallerName, "istio-init", "files", kubernetesCRD)
 	yamlFileName = filepath.Join(k.ReleaseDir, yamlFileName)
 
 	// deploy CRDs first
 	if err := util.KubeApply("kube-system", yamlFileName, k.KubeConfig); err != nil {
 		log.Errorf("Failed to apply %s", yamlFileName)
 		return err
 	}
+	return nil
+}
+
+func (k *KubeInfo) deployIstioWithHelm() error {
+	// Note: When adding a CRD to the install, an entry is needed here as well
+	istioCRDFileNames := []string{"crd-10.yaml", "crd-11.yaml", "crd-certmanager-10.yaml"}
```

Comment:
Hardcoding file names in the code, does not sound right. If these files cannot be passed as arguments, at least consider defining const.

Reply:
go does not have a mechanism to define const slices.  I did try that originally, but go wasn't having it :)

Reply:
You can define string constants and then you them to populate your slice. Also please add comments what these files are, so when people look at the code they can get an idea what is stored in those files and why.

Reply:
done.